Dockerfile.aarch64

```diff
@@ -0,0 +1,20 @@
+FROM resin/aarch64-debian:3.5
```

Comment:
Please compare with https://hub.docker.com/r/aarch64/debian/ (which is an official Docker image)